CONTRIBUTING.md

```diff
@@ -3,24 +3,24 @@
 ## Create the docs locally
 
 ``` python
-conda env update -f environment.yml
+conda env create --file environment.yml
 conda activate geocat-applications
 make clean html
 open _build/html/index.html
```

Comment:
Again, maybe this is or should be handled elsewhere, but I like the language and suggestion from xarray if we're looking for something less macOS specific: https://docs.xarray.dev/en/stable/contributing.html#building-the-documentation

Reply:
I'll add it to #50